app/controllers/miq_ae_class_controller.rb

```diff
@@ -2554,14 +2554,12 @@ def domain_overrides
   end
 
   def get_session_data
-    @layout     = "miq_ae_class"
     @title      = _("Datastore")
```

Comment:
Create a method `title` in this controller and move `_("Datastore")` there. Then you can remove this line :)